{"uid":"97623e57c6fd03b8","name":"Attempt to cancel an already cancelled RTP","fullName":"A Creditor Service Provider can cancel a Request to Pay: Attempt to cancel an already cancelled RTP","historyId":"c5fcd455f64541f905da33d4af1b095f","time":{"start":1759908411151,"stop":1759908411765,"duration":614},"status":"broken","statusMessage":"requests.exceptions.HTTPError: 502 Server Error: Bad Gateway for url: https://api-mcshared.uat.cstar.pagopa.it/auth/token\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"bdd-tests/steps/auth_steps.py\", line 17, in given_ec_on_page\n    access_token = get_valid_access_token(client_id=client_id,\n                                          client_secret=client_secret,\n                                          access_token_function=get_access_token)\n  File \"/home/runner/work/rtp-platform-qa/rtp-platform-qa/api/auth.py\", line 8, in get_valid_access_token\n    token_response.raise_for_status()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/requests/models.py\", line 1026, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\n","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"requests.exceptions.HTTPError: 502 Server Error: Bad Gateway for url: https://api-mcshared.uat.cstar.pagopa.it/auth/token\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"bdd-tests/steps/auth_steps.py\", line 17, in given_ec_on_page\n    access_token = get_valid_access_token(client_id=client_id,\n                                          client_secret=client_secret,\n                                          access_token_function=get_access_token)\n  File \"/home/runner/work/rtp-platform-qa/rtp-platform-qa/api/auth.py\", line 8, in get_valid_access_token\n    token_response.raise_for_status()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/requests/models.py\", line 1026, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\n","steps":[{"name":"Given the debtor A has fiscal code random","time":{"start":1759908411152,"stop":1759908411152,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"And the debtor Service Provider is authenticated","time":{"start":1759908411152,"stop":1759908411764,"duration":612},"status":"broken","statusMessage":"requests.exceptions.HTTPError: 502 Server Error: Bad Gateway for url: https://api-mcshared.uat.cstar.pagopa.it/auth/token\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"bdd-tests/steps/auth_steps.py\", line 17, in given_ec_on_page\n    access_token = get_valid_access_token(client_id=client_id,\n                                          client_secret=client_secret,\n                                          access_token_function=get_access_token)\n  File \"/home/runner/work/rtp-platform-qa/rtp-platform-qa/api/auth.py\", line 8, in get_valid_access_token\n    token_response.raise_for_status()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/requests/models.py\", line 1026, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"And the debtor Service Provider activated the debtor A","time":{"start":1759908411765,"stop":1759908411765,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"And the creditor Service Provider is authenticated","time":{"start":1759908411765,"stop":1759908411765,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Given the creditor Service Provider sent an RTP to the debtor A","time":{"start":1759908411765,"stop":1759908411765,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"And the creditor Service sent a cancellation request for the RTP","time":{"start":1759908411765,"stop":1759908411765,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"When the creditor Service send another cancellation request for the RTP","time":{"start":1759908411765,"stop":1759908411765,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Then the RTP is already cancelled","time":{"start":1759908411765,"stop":1759908411765,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":8,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"tag","value":"authentication"},{"name":"tag","value":"cancel"},{"name":"tag","value":"send"},{"name":"tag","value":"unhappy_path"},{"name":"feature","value":"A Creditor Service Provider can cancel a Request to Pay"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":2,"skipped":0,"passed":9,"unknown":0,"total":11},"items":[{"uid":"cf5fc313e5b4f6c6","reportUrl":"https://pagopa.github.io/rtp-platform-qa/bdd/557//#testresult/cf5fc313e5b4f6c6","status":"passed","time":{"start":1759822227020,"stop":1759822230377,"duration":3357}},{"uid":"288dd3ab5253e8a1","reportUrl":"https://pagopa.github.io/rtp-platform-qa/bdd/556//#testresult/288dd3ab5253e8a1","status":"passed","time":{"start":1759735819514,"stop":1759735822615,"duration":3101}},{"uid":"f9db9ef798f0c291","reportUrl":"https://pagopa.github.io/rtp-platform-qa/bdd/555//#testresult/f9db9ef798f0c291","status":"passed","time":{"start":1759650107956,"stop":1759650111177,"duration":3221}},{"uid":"709175b8bb1208bc","reportUrl":"https://pagopa.github.io/rtp-platform-qa/bdd/554//#testresult/709175b8bb1208bc","status":"passed","time":{"start":1759563796140,"stop":1759563800544,"duration":4404}},{"uid":"6dff9fc0b588fcd2","reportUrl":"https://pagopa.github.io/rtp-platform-qa/bdd/553//#testresult/6dff9fc0b588fcd2","status":"passed","time":{"start":1759484945233,"stop":1759484948642,"duration":3409}},{"uid":"18dcfbf315c59639","reportUrl":"https://pagopa.github.io/rtp-platform-qa/bdd/552//#testresult/18dcfbf315c59639","status":"broken","statusDetails":"requests.exceptions.HTTPError: 502 Server Error: Bad Gateway for url: https://api-mcshared.uat.cstar.pagopa.it/auth/token\n","time":{"start":1759476896162,"stop":1759476896684,"duration":522}},{"uid":"758d9844f7e8041","reportUrl":"https://pagopa.github.io/rtp-platform-qa/bdd/551//#testresult/758d9844f7e8041","status":"passed","time":{"start":1759390891117,"stop":1759390894927,"duration":3810}},{"uid":"c257333858857e79","reportUrl":"https://pagopa.github.io/rtp-platform-qa/bdd/550//#testresult/c257333858857e79","status":"passed","time":{"start":1759316296697,"stop":1759316300500,"duration":3803}},{"uid":"677f2b6f495d12fd","reportUrl":"https://pagopa.github.io/rtp-platform-qa/bdd/549//#testresult/677f2b6f495d12fd","status":"passed","time":{"start":1759303857973,"stop":1759303862561,"duration":4588}},{"uid":"9e6dfd4c4e1710f8","reportUrl":"https://pagopa.github.io/rtp-platform-qa/bdd/548//#testresult/9e6dfd4c4e1710f8","status":"passed","time":{"start":1759217438883,"stop":1759217441936,"duration":3053}}]},"tags":["cancel","send","unhappy_path","authentication"]},"source":"97623e57c6fd03b8.json","parameterValues":[]}